src/system_error.cr

```diff
@@ -85,7 +85,7 @@ module SystemError
     #
     # By default it returns the original message unchanged. But that could be
     # customized based on the keyword arguments passed to `from_errno` or `from_winerror`.
-    protected def build_message(message, **opts) : String
+    protected def build_message(message, **opts) : String?
```

Comment:
Could also add `String?` restriction to `message` like in `from_os_error`.

Reply:
Right. That won't affect functionality but I guess it ends up clearer.